configuring-puppet-enterprise-settings.md

```diff
@@ -1,3 +1,8 @@
+# ⚠ WARNING ⚠
+
+This repository is no longer mantained since ~ 2019. While some of these documentation might still be useful, much of it is obsolete or out of date.
+
+For a maintained collection of similar information, please see [Patterns and Tactics](https://puppet.com/docs/patterns-and-tactics).
```

Comment:
I've noticed in some of the files the warning block mixes with the old content a bit.

@davidsandilands would it be a good idea to add a new line after the Warning block?